react/__tests__/ProductDetails.test.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { renderWithIntl } from 'intl-utils'
+
+import ProductDetails from '../ProductDetails'
+
+describe('<ProductDetails /> component', () => {
+  const getComponentRender = customProps => {
+    const props = {
+      slug: 'productSlug',
+      query: {
+        skuId: 'id',
+      },
+      ...customProps,
+    }
+
+    return renderWithIntl(<ProductDetails {...props} />)
+  }
+
+  it('should match the loading snapshot', () => {
+    const props = {
+      productQuery: {
+        loading: true,
+      },
+    }
+
+    const render = getComponentRender(props)
+    const fragment = render.asFragment()
+    expect(fragment).toMatchSnapshot()
+  })
+
+  it('should match the snapshot with product', () => {
+    const props = {
```

Comment:
what do you think of moving this mock to an external file? It will be more readable, and will keep the test file cleaner.